news/_posts/core-weekly/2021-07-27-coreweekly-29-2021.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 29 of 2021"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2021-07-27
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 19th to Sunday 25th of July 2021.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+In case you missed it, _we moved the next Public OSS Demo session date_ to [**Wednesday, August 4, 2021 at 4pm CEST**](https://www.youtube.com/watch?v=pE6lVtdsIhE). We noticed that 2pm CEST was a little early in the morning for people watching us live from the Americas, so we decided that future sessions will start two hours later — at 4pm. We hope this will encourage our American community to watch the demo live and discuss with us :smile:
```

Comment:
Tricky business: American is usually understood as relating to the United States.
```suggestion
In case you missed it, _we moved the next Public OSS Demo session date_ to [**Wednesday, August 4, 2021 at 4pm CEST**](https://www.youtube.com/watch?v=pE6lVtdsIhE). We noticed that 2pm CEST was a little early in the morning for people watching us live from the Americas, so we decided that future sessions will start two hours later — at 4pm. We hope this will encourage more of our community to watch the demo live and discuss with us :smile:
```